charts/rancher-vsphere-csi/values.yaml

```diff
@@ -144,7 +144,7 @@ versionOverrides:
       csiController:
         image:
           repository: rancher/mirrored-cloud-provider-vsphere-csi-release-driver
-          tag: v3.0.1
+          tag: v3.0.2
```

Comment:
For CSI, it looks like we'll have to wait for 3.1.x for 1.28 support. There are RCs available, but no GA releases, and the vmware docs haven't been updated yet.

Reply:
Sounds like it'd be cleaner to bump both charts to 1.28 at the same time. So, wait?

Reply:
I agree with the above.

Reply:
yeah, might as well wait. thanks!